src/WebJobs.Script.Grpc/Eventing/IGrpcEventManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Wondering why this is commented out and if we have plans to use this in this PR?

Reply:
sorry, must have been leftover from an earlier iteration; removing